### PR TITLE
mui: Change layout Sidebar, InfiniteList, Show and add defaultStyles

### PR DESCRIPTION
**Pull request by killui**

### CODE REVIEW

File: src/mui/detail/Show.js [hunk 1]
```diff
@@ -71,34 +73,37 @@ export class Show extends Component {
...
-                    {data &&
-                        React.cloneElement(children, {
-                            resource,
-                            basePath,
-                            record: data,
-                            translate,
-                            version,
-                        })}
-                </Card>
+          <div style={{ opacity: isLoading ? 0.8 : 1 }}>
```
Comment (Claire-Cleany) [inline src/mui/detail/Show.js:76]: Why excluding this specific style from defaultStyles.js?

File: src/mui/list/InfiniteList.js [hunk 2]
```diff
@@ -117,61 +111,63 @@ export class InfiniteList extends List {
...
-                                    })}
-                            </div>
-                        </InfiniteScroll>
-                    ) : (
-                        <CardText style={styles.noResults}>
-                            {translate('aor.navigation.no_results')}
-                        </CardText>
-                    )}
-                </Card>
+            <div className="list-page" style={{ opacity: isLoading ? 0.8 : 1 }}>
```
Comment (Claire-Cleany) [inline src/mui/list/InfiniteList.js:114]: same question

File: src/mui/detail/Show.js [hunk 3]
```diff
@@ -71,34 +73,37 @@ export class Show extends Component {
...
+            <div style={styles.header}>
```
Comment (thomas-cleany) [inline src/mui/detail/Show.js:77]: you pass styles as a prop line 54, if someone do : 
```
const styles = {
  color: '#fff'',
};

<Show styles={styles} />
```

You will loose all your style. Is that the desired behavior ?

File: src/mui/detail/Show.js [hunk 4]
```diff
@@ -50,6 +51,7 @@ export class Show extends Component {
             hasEdit,
             translate,
             version,
+            styles = defaultStyles,
```
Comment (thomas-cleany) [inline src/mui/detail/Show.js:54]: why use styles and not style ?

Comment (killui) [reply src/mui/detail/Show.js:54]: because styles is an object that contain multiple styles

File: src/mui/detail/Show.js (hunk 3 above)
Comment (killui) [reply src/mui/detail/Show.js:77]: yes, because if we want something specific for a show we can do it

File: src/mui/detail/Show.js (hunk 1 above)
Comment (killui) [reply src/mui/detail/Show.js:76]: I removed this style in my next pr

File: src/mui/list/InfiniteList.js (hunk 2 above)
Comment (killui) [reply src/mui/list/InfiniteList.js:114]: same answer

**Review by Claire-Cleany — APPROVED** [review]

File: src/mui/detail/Show.js (hunk 3 above)
Comment (thomas-cleany) [reply src/mui/detail/Show.js:77]: Hmm ok. so I guess this should have a little documentation because we are expecting for a specific object

**Review by thomas-cleany — APPROVED** [review]

**Pull request closed without merging**